Dockerfile

```diff
@@ -34,6 +29,14 @@ RUN curl -sSLO https://github.com/mozilla/geckodriver/releases/download/${GECKOD
 
 ENTRYPOINT ["poetry", "run", "pytest"]
 
-#Stage 5 - watchdogtest
+#Stage 4 - watchdogtest
 FROM test as watchdogtest
-ENTRYPOINT poetry run watchmedo shell-command --patterns="*.py;*.html" --recursive --command="poetry run pytest"
\ No newline at end of file
+ENTRYPOINT poetry run watchmedo shell-command --patterns="*.py;*.html" --recursive --command="poetry run pytest"
+
+#Stage 5 - production
```

Comment:
This stage needing to be the final step in our Dockerfile in order for our CD pipeline to work is very unintuitive, so it would be a good candidate for a quick comment to that effect; otherwise a future developer is bound to be confused when they break production just by adding a new and seemingly unused stage to this file!